src/combination.jl

```diff
@@ -98,31 +127,123 @@ end
-        combined[j, :, 4] = -(num[:, 1] .+ num[j, 5] .- (num[:, 1] .* num[j, 5]) .- 1.0) .* base # lap w/ palb
-        combined[j, :, 5] = -(num[:, 2] .+ num[j, 3] .- (num[:, 2] .* num[j, 3]) .- 1.0) .* base # dox w/ gem
-        combined[j, :, 6] = -(num[:, 2] .+ num[j, 4] .- (num[:, 2] .* num[j, 4]) .- 1.0) .* base # dox w/ pac
-        combined[j, :, 7] = -(num[:, 2] .+ num[j, 5] .- (num[:, 2] .* num[j, 5]) .- 1.0) .* base # dox w/ palb
-        combined[j, :, 8] = -(num[:, 3] .+ num[j, 4] .- (num[:, 3] .* num[j, 4]) .- 1.0) .* base # gem w/ pac
-        combined[j, :, 9] = -(num[:, 3] .+ num[j, 5] .- (num[:, 3] .* num[j, 5]) .- 1.0) .* base # gem w/ palb
-        combined[j, :, 10] = -(num[:, 4] .+ num[j, 5] .- (num[:, 4] .* num[j, 5]) .- 1.0) .* base # pac w/ palb
+        # the base case for either of combinations is drugA to scale the cell numbers back.
+        combined[j, :, 1] = -(num[:, 1] .+ num[j, 2] .- (num[:, 1] .* num[j, 2]) .- 1.0) .* ((g1s[T, 1, 1] + g2s[T, 1, 1]) + (g1s[T, 1, 2] + g2s[T, 1, 2])) / 2 # lap w/ dox; meaning dox changes with rows and lap changes with columns
+        combined[j, :, 2] = -(num[:, 1] .+ num[j, 3] .- (num[:, 1] .* num[j, 3]) .- 1.0) .* ((g1s[T, 1, 1] + g2s[T, 1, 1]) + (g1s[T, 1, 3] + g2s[T, 1, 3])) / 2 # lap w/ gem
+        combined[j, :, 3] = -(num[:, 1] .+ num[j, 4] .- (num[:, 1] .* num[j, 4]) .- 1.0) .* ((g1s[T, 1, 1] + g2s[T, 1, 1]) + (g1s[T, 1, 4] + g2s[T, 1, 4])) / 2 # lap w/ pac
+        combined[j, :, 4] = -(num[:, 1] .+ num[j, 5] .- (num[:, 1] .* num[j, 5]) .- 1.0) .* ((g1s[T, 1, 1] + g2s[T, 1, 1]) + (g1s[T, 1, 5] + g2s[T, 1, 5])) / 2# lap w/ palb
+        combined[j, :, 5] = -(num[:, 2] .+ num[j, 3] .- (num[:, 2] .* num[j, 3]) .- 1.0) .* ((g1s[T, 1, 2] + g2s[T, 1, 2]) + (g1s[T, 1, 3] + g2s[T, 1, 3])) / 2 # dox w/ gem
+        combined[j, :, 6] = -(num[:, 2] .+ num[j, 4] .- (num[:, 2] .* num[j, 4]) .- 1.0) .* ((g1s[T, 1, 2] + g2s[T, 1, 2]) + (g1s[T, 1, 4] + g2s[T, 1, 4])) / 2 # dox w/ pac
+        combined[j, :, 7] = -(num[:, 2] .+ num[j, 5] .- (num[:, 2] .* num[j, 5]) .- 1.0) .* ((g1s[T, 1, 2] + g2s[T, 1, 2]) + (g1s[T, 1, 5] + g2s[T, 1, 5])) / 2 # dox w/ palb
+        combined[j, :, 8] = -(num[:, 3] .+ num[j, 4] .- (num[:, 3] .* num[j, 4]) .- 1.0) .* ((g1s[T, 1, 3] + g2s[T, 1, 3]) + (g1s[T, 1, 4] + g2s[T, 1, 4])) / 2 # gem w/ pac
+        combined[j, :, 9] = -(num[:, 3] .+ num[j, 5] .- (num[:, 3] .* num[j, 5]) .- 1.0) .* ((g1s[T, 1, 3] + g2s[T, 1, 3]) + (g1s[T, 1, 5] + g2s[T, 1, 5])) / 2 # gem w/ palb
+        combined[j, :, 10] = -(num[:, 4] .+ num[j, 5] .- (num[:, 4] .* num[j, 5]) .- 1.0) .* ((g1s[T, 1, 4] + g2s[T, 1, 4]) + (g1s[T, 1, 5] + g2s[T, 1, 5])) / 2 # pac w/ palb
     end
+
     @assert(all(combined .>= 0.0))
     return combined
 end
 
+
+####--------------- Loewe additivity ------------------####
+""" Find the inverse of a hill function (concentration), given the parameters and the effect. """
+function inv_hill(p::Array{Float64, 1}, y)
```

Comment:
I thought you removed the Loewe additivity? Moving the functions creates a huge diff.

Reply:
Done.